examples/experimental/dagster-airlift/examples/tutorial-example/tutorial_example/dagster_defs/stages/standalone.py

```diff
@@ -20,66 +20,61 @@ def airflow_dags_path() -> Path:
     return Path(__file__).parent / "tutorial_example" / "airflow_dags"
 
 
-def load_csv_to_duckdb_defs(args: LoadCsvToDuckDbArgs) -> Definitions:
-    spec = AssetSpec(key=[args.duckdb_schema, args.table_name])
-
+def load_csv_to_duckdb_defs(spec: AssetSpec, args: LoadCsvToDuckDbArgs) -> Definitions:
     @multi_asset(name=f"load_{args.table_name}", specs=[spec])
     def _multi_asset() -> None:
         load_csv_to_duckdb(args)
 
     return Definitions(assets=[_multi_asset])
 
 
-def export_duckdb_to_csv_defs(args: ExportDuckDbToCsvArgs) -> Definitions:
-    spec = AssetSpec(
```

Comment:
❤️

Reply:
https://linear.app/dagster-labs/issue/FOU-374/eliminate-string-mangling-to-generate-asset-keys-in-export-duckdb-to

feel free to close this one out